apps/openmw/mwmechanics/aisequence.cpp

```diff
@@ -231,7 +232,7 @@ void AiSequence::execute (const MWWorld::Ptr& actor, CharacterController& charac
         if (package->execute (actor,characterController,state,duration))
         {
             // Put repeating noncombat AI packages on the end of the stack so they can be used again
-            if (isActualAiPackage(packageTypeId) && package->getRepeat())
+            if (isActualAiPackage(packageTypeId) && (mRepeat || ((packageTypeId == 0) && package->getRepeat())))
```

Comment:
This was needed because aipackage.cpp was setting it to true by default. Fixing this.